cpp/oneapi/dal/data/graph_service_functions.hpp

```diff
@@ -36,46 +30,46 @@ namespace oneapi::dal::preview {
 
 /// Get number of vertexes in a graph.
 ///
-/// @tparam G  Type of graph
+/// @tparam Graph  Type of graph
 ///
-/// @param [in]   g  graph
+/// @param [in]   graph  Input graph
 ///
-/// @return Result is vertex_size_type of G.
-template <typename G>
-constexpr auto get_vertex_count(const G &g) noexcept -> vertex_size_type<G> {
-    return get_vertex_count_impl(g);
+/// @return Result is vertex_size_type of Graph.
+template <typename Graph>
+constexpr auto get_vertex_count(const Graph& graph) noexcept -> vertex_size_type<Graph> {
+    return get_vertex_count_impl(graph);
 }
 
-template <typename G>
-constexpr auto get_edge_count(const G &g) noexcept -> edge_size_type<G> {
-    return get_edge_count_impl(g);
+template <typename Graph>
+constexpr auto get_edge_count(const Graph& graph) noexcept -> edge_size_type<Graph> {
+    return get_edge_count_impl(graph);
 }
 
 /// Get degree of vertex in a graph.
 ///
-/// @tparam G  Type of graph
+/// @tparam Graph  Type of graph
```

Comment:
'term' type may not be clean and mixed with type in programming sense
What term do we use to describe directed, undirected, represented with adjacency list etc graphs?

Reply:
Documentation will be updated in separate pull request.